test/rebay_api_web/controllers/bid_controller_test.exs

```diff
@@ -75,11 +72,34 @@ defmodule RebayApiWeb.BidControllerTest do
         }
       ] = json_response(conn, 200)["data"]
     end
+
+    test "returns error when user is logged in but request is not authenticated", %{conn: conn, bid: bid} do
+      conn = init_test_session(conn, id: "test_id_token")
+      |> get(Routes.user_bid_path(conn, :index_by_user, bid.user.uuid))
+
+      assert json_response(conn, 401)["errors"] != %{}
+    end
+
+    test "renders error when user is not logged in", %{conn: conn, bid: bid} do
+      conn = conn |> get(Routes.user_bid_path(conn, :index_by_user, bid.user.uuid))
+
+      assert json_response(conn, 401)["errors"] != %{}
+    end
+
+    test "returns error when a logged in user tries to retrieve another users bids", %{conn: conn, bid: bid} do
+      conn = TestHelpers.valid_session(conn, bid.user)
+      |> get(Routes.user_bid_path(conn, :index_by_user, Ecto.UUID.generate()))
+
+      assert json_response(conn, 401)["errors"] != %{}
+    end
```

Comment:
Nice tests 👍

One thing I would point out as an outsider: the differences between the connection states between the tests using `init_test_session` and `TestHelpers.valid_session` aren't very clear to me without digging into what each of those functions do. I think `TestHelpers.invalid_session` or something would be a nice tweak here that would make it easier to skim.